### DISCUSSION TITLE
Backward reaction rates

### DISCUSSION THREAD

**Discussion by VirgileCharton** [mutationpp/Mutationpp#206] [General]
Hi,

I have a question about backward reaction rates. Is there a way to specify them by hand in the mechanism file, or does M++ not allow that behavior to prevent possible issues and always compute them from the equilibrium constants ? I have Arrhenius set of coefficients for direct and inverse reactions that are used for a CFD calculation and I would like to explicitly write the two reactions ways when using M++.

I am a new M++ user so I might have missed something in the docs.

Thank you very much.

Regards,

Virgile Charton

**Comment by capriatim** [comment]
Hi @VirgileCharton ,

have you tried to define for each reaction two separate irreversible reactions ('=>' in spite of '=')?
https://github.com/mutationpp/Mutationpp/blob/master/docs/input-files.md#reaction-mechanisms

Best regards. Michele
